src/sentry/models/outbox.py

```diff
@@ -35,6 +35,7 @@ class OutboxScope(IntEnum):
     WEBHOOK_SCOPE = 2
     AUDIT_LOG_SCOPE = 3
     USER_IP_SCOPE = 4
+    INTEGRATION_SCOPE = 5
```

Comment:
For control silo outboxes, generally the relevant scopes are 
1.  Users (we wrote a user object, and sync it out)
2.  Integrations (we wrote an integration change, and sync it out)

Scopes define ordering of processing, as explained in the Outbox model comments.  It is fine if per user and per integration processing is not ordered within a single identifier.